decredPoliteia/decredPoliteia_vo.md

```diff
@@ -3,29 +3,27 @@
 
 Named after the Greek term meaning ‘a system of government’--Politeia is the Decred proposal system: a chain-anchored social media application where stakeholders decide which projects recieve funding from the network fund. 
 
-Decred governance data hosted on Politeia is made up of submitted project proposals, comments, and votes. 
+Decred governance data hosted on Politeia is made up of submitted project proposals, comments, and up/down votes. 
 
-This user-sourced data is episodically anchored to the Decred blockchain creating an immutable and cryptographically verifiable public record of Decred self-governance. 
-
-This means the data stored in Politeia can never be altered, deleted, or forged--ensuring that attempts to manipulate Decred's governance are significantly more difficult than if a traditional website or social media application were in use.
+This user-sourced content is episodically anchored to the Decred blockchain. Censorship tokens allow users to demonstrate when their proposals or comments have been censored.
 
 To achieve this, Politeia's off-chain storage system makes use of Git (a popular revision control system), and DecredTime (a custom server application for storing timestamps on the Decred blockchain).
 
 Git is utilized to create a historic record of every change to the data stored on Politeia. All revisions recorded by Git are anchored to the Decred blockchain through DecredTime. This "Git plus timestamping" equation results in a time sequence of data revisions that is computationally infeasible to recreate.
 
 On the front end, the Politeia GUI provides an easy to use web-based interface for stakeholders to submit project proposals, comments, and votes.
 
-In order to limit the amount of on-chain storage required, low quality proposals and spam posts are weeded out through a human review process.
+There is a small DCR fee for registering new Politeia accounts (to limit sock-puppets), and for submitting proposals (to limit spam).
```

Comment:
I think the limiting of on-chain storage is a good bonus to mention. Also important to add mention of Fees.